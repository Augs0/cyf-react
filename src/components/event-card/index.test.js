import React from 'react';
import { shallow } from 'enzyme';

import EventCard from './';

describe('EventCard Component', () => {
  it('renders correctly', () => {
    const mockEvent = {
      city: 'London',
      description: 'First bit of javascript',
      endDate: '1515851599504',
      eventId: '1',
      intake: 'London Class 3',
      links: [
        {
          'course materials':
            'https://codeyourfuture.github.io/syllabus-master/js-core/week-04/lesson.html',
        },
      ],
      location: 'Ticketmaster',
      mentors: [
        { name: 'Matt', avatar: 'www.something.com' },
        { name: 'Jun', avatar: 'www.somthing.com' },
        { name: 'Felix', avatar: 'www.somthing.com' },
      ],
      moduleLeaders: [{ name: 'Kash', avatar: 'something' }],
      startDate: '1515851599504',
      topic: 'Week 4 - Hello Javascript',
    };

    expect(shallow(<EventCard {...mockEvent} />)).toMatchSnapshot();
  });
});
