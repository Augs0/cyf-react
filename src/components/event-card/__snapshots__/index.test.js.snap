// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventCard Component renders correctly 1`] = `
<Styled(div)>
  <Styled(p) />
  <p>
    First bit of javascript
  </p>
  <Styled(div)>
    <p>
      <span>
        Ticketmaster
      </span>
    </p>
    <p>
      <span>
        March 18, 2018
      </span>
      <span>
         
      </span>
      <span>
         - 
      </span>
      <span />
    </p>
  </Styled(div)>
  <Styled(a)
    href="/events/1"
  >
    View event
  </Styled(a)>
</Styled(div)>
`;
